README.md

```diff
@@ -56,26 +56,26 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 	+ Install [Ruby](https://www.ruby-lang.org/en/):
       ```
-      $ rbenv install 2.3.1
+      $ rbenv install 2.5.3
```

Comment:
I used to have `rbenv` as pkg manager, this didn't work - they may have changed the naming